index.ts

```diff
@@ -3,7 +3,7 @@ import { context } from '@actions/github';
 import { Client } from 'asana';
 
 
-const ASANA_TASK_LINK_REGEX = /https:\/\/app.asana.com\/(\d+)\/(?<project>\d+)\/(?<taskId>\d+).*?/ig;
+const ASANA_TASK_LINK_REGEX = /https:\/\/app\.asana\.com\/(?:(?:\d+\/home\/\d+\/)|(?:\d+\/\d+\/project\/\d+\/task\/))(?<taskId>\d+)/ig;
```

Comment:
This doesn't look right. I'm still getting the old URL format when copying the link from most tasks:

https://app.asana.com/0/1208695219574823/1209624742730486
https://app.asana.com/0/1209334705014692/1208935854349560/f
https://app.asana.com/0/0/1209295166396317/f

Where are you seeing this new url format?

Reply:
The updated version should now capture it I think?